lib/std/fs/path.zig

```diff
@@ -1110,7 +1110,7 @@ pub fn relativePosix(allocator: *Allocator, from: []const u8, to: []const u8) ![
         }
         if (to_rest.len == 0) {
             // shave off the trailing slash
-            return result[0 .. result_index - 1];
+            return allocator.shrink(result, result_index - 1);
```

Comment:
Code needs to shrink the buffer so that the length is correct when free is called on the buffer later.

Reply:
This doesn't need to block the merge, but this would be an excellent source code comment rather than GitHub comment